corehq/sql_accessors/sql_templates/soft_undelete_cases.sql

```diff
@@ -0,0 +1,18 @@
+DROP FUNCTION IF EXISTS soft_undelete_cases(TEXT, TEXT[]);
+
+CREATE FUNCTION soft_undelete_cases(
+    p_domain TEXT,
+    case_ids TEXT[],
+    affected_count OUT INTEGER) AS $$
+BEGIN
+    UPDATE form_processor_commcarecasesql SET
+        deleted = FALSE,
+        server_modified_on = p_server_modified_on,
+        deletion_id = NULL,
```

Comment:
ouf that makes me worried that my tests were not testing this properly. good catch 🐦